ramls/inventory.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 title: Inventory API
-version: v11.0
+version: v11.1
```

Comment:
The version here is 11.1 and in the module descriptor it is 11.2

Reply:
I've set it so that both versions are 11.2